fetch/main.js

```diff
@@ -0,0 +1,7 @@
+fetch('https://pokeapi.co/api/v2/pokemon/69')
+  .then((data) => {
+    return data.json();
+  })
+  .then((pokedata) => {
+    console.log(pokedata.name, pokedata);
+  });
```

Comment:
Since `fetch` requests always have a chance to fail, please make sure you're always handling that failure.